src/main/java/edu/cornell/kfs/tax/FormTypes1099.java

```diff
@@ -0,0 +1,32 @@
+package edu.cornell.kfs.tax;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+import org.apache.commons.lang3.builder.ToStringStyle;
+
+public enum FormTypes1099 {
+	MISC("MISC", "1099-Misc"),
+	NEC("NEC", "1099-NEC");
+	
+	
+	public final String formCode;
+	public final String formDescription;
+	
+	private FormTypes1099 (String formCode, String formDescription) {
+		this.formCode = formCode;
+		this.formDescription = formDescription;
```

Comment:
I'm noticing a lot of tab characters in the new and modified code sections of this PR's files. Can these tabs be converted into spaces instead?

Reply:
ok